### PR TITLE
Update Winapp2.ini

### PR DESCRIPTION
**Pull request by zero77**
Cleans up files that are created and left behind by chocolatey.

### CODE REVIEW

File: Winapp2.ini [hunk 1]
```diff
@@ -57,6 +57,16 @@ RegKey1=HKCU\Software\Chromium\BrowserExitCodes
 RegKey2=HKCU\Software\CocCoc\Browser\BrowserExitCodes
 RegKey3=HKCU\Software\Google\Chrome\BrowserExitCodes
 
+[Chocolatey *]
+LangSecRef/Section=3024
```
Comment (MoscaDotTo) [inline Winapp2.ini]: Only LangSecRef is needed here, combining two keys is a syntax error

File: Winapp2.ini [hunk 2]
```diff
@@ -57,6 +57,16 @@ RegKey1=HKCU\Software\Chromium\BrowserExitCodes
...
+Detect/DetectFile=%CommonAppData%\chocolatey
```
Comment (MoscaDotTo) [inline Winapp2.ini]: As above, if defining a path on the file system, use only DetectFile

File: Winapp2.ini [hunk 3]
```diff
@@ -57,6 +57,16 @@ RegKey1=HKCU\Software\Chromium\BrowserExitCodes
...
+Default=False
+FileKey1=%CommonAppData%\chocolatey\logs\choco.summary.log
+FileKey2=%CommonAppData%\chocolatey\logs\chocolatey.log
+FileKey3=%CommonAppData%\chocolatey\lib\*\*.txt
+FileKey4=%LocalAppData%\Temp\chocolatey\|*.*|REMOVESELF
```
Comment (MoscaDotTo) [inline Winapp2.ini]: Keys needn't include %LocalAppData%\Temp\.. keys as it's presumed that whichever cleaner is running this entry will empty the temp folder anyway

**Comment by zero77** [comment]
@MoscaDotTo 
Thanks for the constructive feedback but, it should be ok now.

File: Winapp2.ini [hunk 4]
```diff
@@ -61,9 +61,8 @@ RegKey3=HKCU\Software\Google\Chrome\BrowserExitCodes
 LangSecRef=3024
 DetectFile=%CommonAppData%\chocolatey
 Default=False
-FileKey1=%CommonAppData%\chocolatey\logs\choco.summary.log
-FileKey2=%CommonAppData%\chocolatey\logs\chocolatey.log
-FileKey3=%CommonAppData%\chocolatey\lib\*\*.txt
+FileKey1=%CommonAppData%\chocolatey\logs|*.log
+FileKey2=%CommonAppData%\chocolatey\lib\*\*.txt
```
Comment (MoscaDotTo) [inline Winapp2.ini]: Almost missed this! :) Should be |*.txt

**Pull request merged**